connector/exceptionsconnector/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+type: exceptions
+
+status:
+  class: connector
+  stability:
+    alpha: [traces_to_metrics, traces_to_logs]
```

Comment:
Given that there are open tasks after this PR gets merged, especially the span.id and trace.id, I would change the stability of this to "development" instead.